source/about/frequently-asked-questions.rst

```diff
@@ -355,14 +355,14 @@ No, if you have already signed the `Mattermost CLA <https://mattermost.org/matte
 Can I compile your plugins by myself?
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-Yes. If you have a Mattermost Enterprise E20 license, you are free to compile and use a plugin under the Mattermost Source Available License. Furthermore, if you are developing against or testing with such a plugin, you are free to compile and test a plugin even without a Mattermost Enterprise E20 license. Without an Enterprise E20 license, source available plugins may have reduced functionality or refuse to start altogether. Request a `trial license <https://mattermost.com/trial/>`_ if your testing requires access to enterprise functionality.
+Yes. If you have a Mattermost Enterprise Edition license, you are free to compile and use a plugin under the Mattermost Source Available License. Furthermore, if you are developing against or testing with such a plugin, you are free to compile and test a plugin even without a Mattermost Enterprise Edition license. Without an Enterprise Edition license, source available plugins may have reduced functionality or refuse to start altogether. Request a `trial license <https://mattermost.com/trial/>`_ if your testing requires access to enterprise functionality.
```

Comment:
Maybe the solution to mentioning the vague "Enterprise" license (E20, Professional, Enterprise) throughout our docs is to always have a note at the top of the page explaining "terms" on the page. This includes things such as the "Enterprise" license.

It's important to note that individual features are available to some but not all Enterprise licenses. But in general, you need to have the appropriate license to _run_ the code, but you don't need a license to __read__ the code.

Reply:
Great suggestion! We're introducing a new system to explain this information to readers so that should be covered soon. :)